MODULE.bazel

```diff
@@ -2,23 +2,22 @@
 
 module(
     name = "com_bookingcom_rules_pitest",
-    version = "0.0.0",
-    compatibility_level = 1,
+    compatibility_level = 6,
```

Comment:
You don't need this change, this gets replaced at release time

Reply:
ok